x-pack/plugins/apm/server/lib/service_map/get_service_anomalies.ts

```diff
@@ -39,29 +39,13 @@ export async function getServiceAnomalies({
     throw Boom.notImplemented(ML_ERRORS.ML_NOT_AVAILABLE);
   }
 
-  const mlCapabilities = await ml.mlSystem.mlCapabilities();
-
-  if (!mlCapabilities.mlFeatureEnabledInSpace) {
-    throw Boom.forbidden(ML_ERRORS.ML_NOT_AVAILABLE_IN_SPACE);
-  }
```

Comment:
should we also remove this check from the other places as well? (`createAnomalyDetectionJobs`, `getAnomalyDetectionJobs`, `hasLegacyJobs`)

Reply:
I'll leave those as-is for now as any performance issues there are less of an issue, but definitely something we should look at together with the ML team in future releases.